src/Views/SuppliesView.vala

```diff
@@ -1,72 +1,141 @@
-/*-
- * Copyright (c) 2015-2018 elementary LLC. (https://elementary.io)
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Library General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Library General Public License for more details.
- *
- * You should have received a copy of the GNU Library General Public
- * License along with this library; if not, write to the
- * Free Software Foundation, Inc., 51 Franklin Street - Fifth Floor,
- * Boston, MA 02110-1301, USA.
+/*
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ * SPDX-FileCopyrightText: 2015-2023 elementary, Inc. (https://elementary.io)
  *
  * Authored by: Corentin Noël <corentin@elementary.io>
  */
 
-public class Printers.SuppliesView: Gtk.ScrolledWindow {
+public class Printers.SuppliesView: Gtk.Box {
     public Printer printer { get; construct; }
 
+    private const string STYLE_CLASS =
+    """
+    block.filled {
+        background-color: #%s;
+    }
+    """;
+
     public SuppliesView (Printer printer) {
         Object (printer: printer);
     }
 
     construct {
-        var name_label = new Gtk.Label (_("Description:"));
-        name_label.xalign = 1;
+        var name_entry = new Gtk.Entry () {
+            hexpand = true,
+            placeholder_text = _("BrandPrinter X3000")
+        };
+
+        var name_label = new Granite.HeaderLabel (_("Description")) {
+            mnemonic_widget = name_entry
```

Comment:
Currently has no effect? I presume a mnemonic will be set after porting to Granite-7.